fibsem/milling/core.py

```diff
@@ -19,11 +19,33 @@
 
 ########################### SETUP
 
+def get_stage_reference_image(
+    microscope: FibsemMicroscope, milling_stage: FibsemMillingStage
+) -> FibsemImage:
+    ref_image = milling_stage.reference_image
```

Comment:
There should only be two paths through here: 
1. return the previously acquired image
2. If image is None, acquire a new image
See comment above above overloaded variables